src/List/ListItem.js

```diff
@@ -5,8 +5,8 @@ import ColorManipulator from '../utils/colorManipulator';
 import transitions from '../styles/transitions';
 import EnhancedButton from '../internal/EnhancedButton';
 import IconButton from '../IconButton';
-import OpenIcon from '../svg-icons/navigation/arrow-drop-up';
-import CloseIcon from '../svg-icons/navigation/arrow-drop-down';
+import OpenIcon from '../svg-icons/navigation/keyboard-arrow-up';
+import CloseIcon from '../svg-icons/navigation/keyboard-arrow-down';
```

Comment:
Oh. I didn't knowthey existed. I can change it. 

My bad. Let me make the changes. Thanks for the update @oliviertassinari